modules/tasks/tasks/src/main/java/org/motechproject/tasks/service/impl/ChannelServiceImpl.java

```diff
@@ -177,6 +181,49 @@ protected void doInTransactionWithoutResult(TransactionStatus status) {
         return channels;
     }
 
+    private void registerLoggerChannel() {
```

Comment:
Ticket said to built it in tasks channel, anyway, changed it according to your suggestions